examples/stickysession/nginx/sticky-ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: nginx-test
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/sticky-enabled: "true"
+    ingress.kubernetes.io/sticky-name: "route"
+    ingress.kubernetes.io/sticky-hash: "sha1"
```

Comment:
I assume you picked sha1 because there's a performance tradeoff here? If we really care about collisions, we should use sha256 at least, if we don't, we can probably get away with md5, its faster (128 bits vs 160)